css/styles.css

```diff
@@ -62,6 +67,21 @@ h5 {
     padding: 0 5px;
 }
 
+a {
+    background-color: #4CAF50;
+    color: white;
```

Comment:
A named colour is inconsistent with all the other hex values.

Reply:
Fixed